### ISSUE TITLE
Make use of space bar

### ISSUE THREAD

**Issue by flit** [flit/MidiKeys#16]
There's a big button waiting to be used.

Ideas:

- sustain
- velocity change/max

**Comment by kevleyski** [comment]
Yeah Sustain would be good - glancing at the code there is no 64 control code sent today? I might add this
